doc/classes/AStar2D.xml

```diff
@@ -143,6 +143,7 @@
 			<description>
 				Returns an array with the IDs of the points that form the path found by AStar2D between the given points. The array is ordered from the starting point to the ending point of the path.
 				If there is no valid path to the target, and [param allow_partial_path] is [code]true[/code], returns a path to the point closest to the target that can be reached.
+				[b]Note:[/b] When [param allow_partial_path] is [code]true[/code] and the destination is a disabled point the execution of this function can be long!
```

Comment:
```suggestion
				[b]Note:[/b] When [param allow_partial_path] is [code]true[/code] and the destination is a disabled point the search can take an unusually long time to finish.
```

Reply:
Nitpick, use "may" for "has the possibility of"
```suggestion
				[b]Note:[/b] When [param allow_partial_path] is [code]true[/code] and [param to_id] is disabled the search may take an unusually long time to finish.
```

Reply:
Excellent suggestions for the docs as always. I kept the use of "solid" instead of disabled for the AStarGrid2D though as for whatever reason in that class it uses "solid" instead of "disabled"!